src/tables.js

```diff
@@ -9,6 +9,8 @@ const Flow = {
   STEM_HEIGHT: 35,
   STAVE_LINE_THICKNESS: 1,
   RESOLUTION: 16384,
+  TABLATURE_FONT: '10pt Arial',
+  TABLATURE_GRACE_FONT: '7.5pt Arial',
```

Comment:
Can you move these constants to `tabnote.js` and `gracetabnote.js`? I'd like to keep the configurables close to the classes.